devtools/server/actors/replay/connection.js

```diff
@@ -649,6 +657,7 @@ function setRecordingFinished(browser, url) {
   if (!url) {
     const contentUrl = browser.currentURI.spec;
 
+    pingTelemetry("recording", "finished", { action: "reloadContentUrl", recordingState });
```

Comment:
it's hard for me to know as a reader, but will we be able to get Median and P90 timing scores from these or do we need additional fields with timing data when this finishes.

Also, is it obvious which event corresponds with a new tab being created and what took the most time...

Reply:
I'm not sure to start extent that can be derived. With this PR, we'll have several events for each button press so the data will exist. However, if we need to include a calculated duration because we can't correlate or calculate it in honeycomb, we can add that by extending the recording state object to include some timestamps.

Reply:
@ryanjduffy i think it is probably easier to calculate the durations here. The general rule of thumb for honeycomb is that it prefers fewer events with more fields

Reply:
That was @jazzdan 's feedback as well :) Adding that shortly